src/main.rs

```diff
@@ -28,6 +28,36 @@ struct Cli {
     command: Commands,
 }
 
+#[derive(clap::Parser, ValueEnum, Debug, Clone, PartialEq, Eq)]
+enum CompletionShell {
+    Bash,
+    Elvish,
+    Fish,
+    Nushell,
+    PowerShell,
```

Comment:
The variant should be named `Powershell` to prevent `-` from being added to the `<SHELL>` arg value.

Reply:
Will you create a PR for the fix?

Reply:
Looking at clap_complete the enum is also named `PowerShell` https://docs.rs/clap_complete/latest/clap_complete/aot/enum.Shell.html